go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/testify v1.4.0 // indirect
 	golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa // indirect
-	golang.org/x/sys v0.0.0-20200202164722-d101bd2416d5 // indirect
+	golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae // indirect
```

Comment:
Do we need these dependency updates?

Reply:
Not sure we do the git-workflow vendor check failed at one stage and did a  `make vendor/fix` to get by it. could revert and see if it passes again .

Reply:
We think it's related to some testing package which was introduced but then removed. The vendor changes are removed now and all is still working. 👍